mars/optimizes/jax.py

```diff
@@ -0,0 +1,32 @@
+from .optimizer import Optimizer
```

Comment:
Apache 2 license should be prepended. Rename the file to avoid relative import behavior in Python 2.